src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -181,6 +218,59 @@ func (repo *MarketplaceCmdRepo) runCmdSteps(
 	return nil
 }
 
+func (repo *MarketplaceCmdRepo) updateFilesOwnershipAndPermissions(
+	installDir valueObject.UnixFilePath,
+) error {
+	installDirStr := installDir.String()
+	_, err := infraHelper.RunCmdWithSubShell(
+		"chown -R nobody:nogroup -L " + installDirStr,
+	)
+	if err != nil {
+		return errors.New(
+			"UpdateInstalledFilesOwnershipError (" + installDirStr + "): " +
+				err.Error(),
+		)
+	}
+
+	_, err = infraHelper.RunCmdWithSubShell(
+		`find ` + installDirStr + ` -type d -exec chmod 755 {} \; && find ` +
+			installDirStr + ` -type f -exec chmod 644 {} \;`,
+	)
+	if err != nil {
+		return errors.New(
+			"UpdateInstalledDirsAndFilesPermissionsError (" + installDirStr + "): " +
+				err.Error(),
+		)
+	}
+
+	return nil
+}
```

Comment:
Essas mensagens de erro provavelmente serão enviadas para a próxima camada com um log.Printf, logo, esse prefixo gigantesco é inútil. Você pode usar ChownError e ChmodError pois lá no método principal que chama essa função muito provavelmente você já botou um prefixo chamado UpdateFileOwnershipAndPermissions (nome gigante btw, novamente, verbosidade excessiva).

A intenção do método é garantir que os arquivos e diretórios tenham as permissões e posse corretas, portanto, updateFilePrivileges seria sucinto o suficiente por exemplo.

Reply:
Ajustado.